tests/test_readline_shell.py

```diff
@@ -23,3 +23,22 @@ def test_render_completions(prefix, completion, prefix_len, readline_completion)
     assert _render_completions({completion}, prefix, prefix_len) == [
         readline_completion
     ]
+
+
+@pytest.mark.parametrize(
+    "line, exp",
+    [
+        [repr("hello"), None],
```

Comment:
Why won't this show hello?

Reply:
it shows hello. instead of repeating, i just use the line again

Reply:
It is for it to be concise, anyway I just refactored it.